packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -1,49 +1,164 @@
-Requires: %{?scl_prefix}rubygem(dynflow) < 2.0
-Requires: %{?scl_prefix}rubygem(get_process_mem)
-Requires: %{?scl_prefix}rubygem(parse-cron) >= 0.1.4
-Requires: %{?scl_prefix}rubygem(parse-cron) < 0.2.0
+Requires: %{?scl_prefix}rubygem(dynflow) >= 1.2.2
 Requires: %{?scl_prefix_ror}rubygem(sinatra)
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}rubygems
-
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: %{?scl_prefix}rubygem(parse-cron) >= 0.1.4
+Requires: %{?scl_prefix}rubygem(parse-cron) < 0.2
+Requires: %{?scl_prefix}rubygem(get_process_mem)
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks-core)
-BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 1.2.1
-BuildRequires: %{?scl_prefix}rubygem(dynflow) < 2.0
-BuildRequires: %{?scl_prefix}rubygem(get_process_mem)
-BuildRequires: %{?scl_prefix}rubygem(parse-cron) >= 0.1.4
-BuildRequires: %{?scl_prefix}rubygem(parse-cron) < 0.2.0
+BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 1.2.2
 BuildRequires: %{?scl_prefix_ror}rubygem(sinatra)
-BuildRequires: foreman-plugin >= 1.17.0
-BuildRequires: foreman-assets
-
+BuildRequires: %{?scl_prefix}rubygem(parse-cron) >= 0.1.4
+BuildRequires: %{?scl_prefix}rubygem(parse-cron) < 0.2
+BuildRequires: %{?scl_prefix}rubygem(get_process_mem)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 %{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-plugin-%{plugin_name} = %{version}
+# end specfile generated dependencies
+
+# start package.json devDependencies BuildRequires
+#BuildRequires: npm(@storybook/addon-actions) >= 5.0.1
+#BuildRequires: npm(@storybook/addon-actions) < 6.0.0
+#BuildRequires: npm(@storybook/addon-knobs) >= 5.0.1
+#BuildRequires: npm(@storybook/addon-knobs) < 6.0.0
+#BuildRequires: npm(@storybook/react) >= 5.0.1
+#BuildRequires: npm(@storybook/react) < 6.0.0
+BuildRequires: npm(babel-cli) >= 6.10.1
+BuildRequires: npm(babel-cli) < 7.0.0
+BuildRequires: npm(babel-core) >= 6.26.3
+BuildRequires: npm(babel-core) < 7.0.0
+#BuildRequires: npm(babel-eslint) >= 8.2.3
+#BuildRequires: npm(babel-eslint) < 9.0.0
+#BuildRequires: npm(babel-jest) >= 23.6.0
+#BuildRequires: npm(babel-jest) < 24.0.0
+BuildRequires: npm(babel-loader) >= 7.1.1
+BuildRequires: npm(babel-loader) < 8.0.0
+#BuildRequires: npm(babel-plugin-dynamic-import-node) >= 2.0.0
+#BuildRequires: npm(babel-plugin-dynamic-import-node) < 3.0.0
+BuildRequires: npm(babel-plugin-lodash) >= 3.3.4
+BuildRequires: npm(babel-plugin-lodash) < 4.0.0
+BuildRequires: npm(babel-plugin-module-resolver) >= 3.2.0
```

Comment:
Looks like this still needs packaging.

Reply:
https://github.com/theforeman/foreman-packaging/pull/3704